packages/headless/src/app/headless-engine.ts

```diff
@@ -281,6 +291,11 @@ export class HeadlessEngine<Reducers extends ReducersMapObject>
     }
   }
 
+  public addReducers(reducers: ReducersMapObject) {
+    this.reducerManager.add(reducers);
+    this.reduxStore.replaceReducer(this.reducerManager.reducer);
```

Comment:
Small nitpick, but shouldn't the reducer manager be in charge of calling `replaceReducer` internally on the redux store, when `add` is executed ?

Seems natural to me that a manager would also manage that part.

Reply:
Hmm, I see ReducerManager's role as taking care of holding, adding and building the final `reducer` function.

I feel giving it the role of replacing the reducer on the store would be a tad too much. The `reduxStore` instance is part of this class. I could pass it into ReducerManager, but I feel it would be clearer to not do that,